apps/ex_nvr_web/priv/static/openapi.yaml

```diff
@@ -611,6 +611,19 @@ components:
             The mountpoint/folder where to store device data (recordings, bif, ...etc.)
 
             This setting is not updatable after the creation of the device
+        override_on_full_disk:
+          type: boolean
+          default: false
+          description: A boolean value to signal that when reaching the override_on_full_disk_threshold on the storage_address disk usage percentage some old recordings would be deleted
```

Comment:
You need to add more details here. Like when the process is triggered, how many recordings are deleted, ..etc.